Products/PasswordStrength/__init__.py

```diff
@@ -1,26 +1,26 @@
-
-"""PasswordStrength
-"""
-
-__author__ = "Dylan Jay <software@pretaweb.com"
-
-from AccessControl.Permissions import add_user_folders
-from Products.PluggableAuthService import registerMultiPlugin
-from plugin import PasswordStrength, \
-                   manage_addPasswordStrength, \
-                   manage_addPasswordStrengthForm
-
-def initialize(context):
-    """Initialize the PasswordStrength plugin."""
-    registerMultiPlugin(PasswordStrength.meta_type)
-    
-    context.registerClass(PasswordStrength,
-                          permission=add_user_folders,
-                          constructors=(manage_addPasswordStrengthForm,
-                                        manage_addPasswordStrength),
-                          #icon='www/noduplicatelogin.png',
-                          visibility=None,
-                          )
-
-
-PROJECTNAME = "PasswordStrength"
+
+"""PasswordStrength
+"""
+
+__author__ = "Dylan Jay <software@pretaweb.com"
+
+from AccessControl.Permissions import add_user_folders
+from Products.PluggableAuthService import registerMultiPlugin
+from zope.i18nmessageid import MessageFactory
```

Comment:
There are pep8 violations here, which has requirements for import ordering: Group standard lib imports, third party module imports, and internal imports. Sort alphabetically within each group.

It is probably worth running all of this code through pep8 and frosted.